shared/AppInsightsCore/Tests/Unit/src/CookieManager.Tests.ts

```diff
@@ -266,39 +309,61 @@ export class CookieManagerTests extends AITestClass {
                     enabled: undefined,
                     ignoreCookies: undefined,
                     blockedCookies: undefined,
-                    getCookie: neverCalled,
-                    setCookie: neverCalled,
-                    delCookie: neverCalled
+                    getCookie: getCookieFn,
+                    setCookie: setCookieFn,
+                    delCookie: delCookieFn
                 }, core.config.cookieCfg);
             }
         });        
 
         this.testCase({
             name: "CookieManager: disable cookies using legacy and new setting both enabled",
             test: () => {
+                let getCookieCallCount = 0;
```

Comment:
Any tests which is only expecting 0 number of calls and never enables the cookie manager should continue to use the previous pattern -- revert this change in this test